### PR TITLE
remove urllib3 pin

### PR DESCRIPTION
**Pull request by Bubu**
requests 2.22.0 as been released supporting urllib3 1.25.2

### Pull Request Checklist

* [x] Pull request is based on the develop branch
* [x] Pull request includes a [changelog file](https://github.com/matrix-org/synapse/blob/master/CONTRIBUTING.rst#changelog)
* [x] Pull request includes a [sign off](https://github.com/matrix-org/synapse/blob/master/CONTRIBUTING.rst#sign-off)

### CODE REVIEW

File: changelog.d/5230.misc [hunk 1]
```diff
@@ -0,0 +1 @@
+Remove urllib3 pin as requests 2.22.0 as been released supporting urllib3 1.25.2
```
Comment (anoadragon453) [inline changelog.d/5230.misc]:
```suggestion
Remove urllib3 pin as requests 2.22.0 as been released supporting urllib3 1.25.2
```

Pedantic, I know :)

Comment (Bubu) [reply changelog.d/5230.misc]: I actually thought about this for a second, didn't know what to do and decided against adding the dot. :D

Buuut there is actually a type in there s/as been/has been/.

Comment (anoadragon453) [reply changelog.d/5230.misc]: Oh yeah huh.

Jokes on you though, we require the dot >:D https://github.com/matrix-org/synapse/blob/master/CONTRIBUTING.rst#changelog

Comment (Bubu) [reply changelog.d/5230.misc]: now fixed.

Comment (Bubu) [reply changelog.d/5230.misc]: You mean I should read more than the first paragraph of contribution guideline docs? :thinking: ;-)

**Pull request merged**